docs/configuration_options.md

```diff
@@ -71,7 +71,7 @@
 | **<a href="#user-content-reuse_cluster" name="reuse_cluster">reuse_cluster</a>**  | If reuse_cluster is set it should hold test_id of the cluster that will be reused.<br>`reuse_cluster: 7dc6db84-eb01-4b61-a946-b5c72e0f6d71` | N/A | SCT_REUSE_CLUSTER
 | **<a href="#user-content-test_id" name="test_id">test_id</a>**  | test id to filter by | N/A | SCT_TEST_ID
 | **<a href="#user-content-db_nodes_shards_selection" name="db_nodes_shards_selection">db_nodes_shards_selection</a>**  | How to select number of shards of Scylla. Expected values: default/random.<br>Default value: 'default'.<br>In case of random option - Scylla will start with different (random) shards on every node of the cluster | default | SCT_NODES_SHARDS_SELECTION
-| **<a href="#user-content-seeds_selector" name="seeds_selector">seeds_selector</a>**  | How to select the seeds. Expected values: reflector/random/first | first | SCT_SEEDS_SELECTOR
+| **<a href="#user-content-seeds_selector" name="seeds_selector">seeds_selector</a>**  | How to select the seeds. Expected values: random/first/all | all | SCT_SEEDS_SELECTOR
```

Comment:
Perhaps it would be better to add also a note that this parameter is a legacy/obsolete/not required from scylla side anymore.

Reply:
This option influences the `scylla.yaml`'s `seeds` option values directly.
So, it unclear to me in which way it is `obsolete`. It still does the same function as before, just changed the set of acceptable config option values.

Reply:
i agree, cause we don't use `reflector` for the seeds, but the parameter is still mandatory in `scylla.yaml`